src/lib/php/Dao/CopyrightDao.php

```diff
@@ -70,13 +70,19 @@ public function getHighlights($uploadTreeId, $tableName="copyright", $agentId=0,
     }
 
     $statementName = __METHOD__.$tableName;
-    $sql = "SELECT * FROM $tableName WHERE copy_startbyte IS NOT NULL AND pfile_fk=$1";
     $params = array($pFileId);
-    if($agentId!=0) {
+    $addAgentValue = "";
+    if(!empty($agentId)) {
       $statementName .= '.agentId';
-      $sql .= ' AND agent_fk=$2';
+      $addAgentValue = ' AND agent_fk=$2';
       $params[] = $agentId;
     }
+    $getHighlightForTableName = "SELECT * FROM $tableName WHERE copy_startbyte IS NOT NULL AND pfile_fk=$1$addAgentValue";
```

Comment:
minor inconsistency:
here you write `$1$addAgentValue`
below you write `$1 $addAgentValue`